src/services/quotes/source-lists/types.ts

```diff
@@ -7,6 +7,7 @@ import { QuoteRequest, SourceMetadata, SourceId, GlobalQuoteSourceConfig, QuoteT
 export type IQuoteSourceList = {
   supportedSources(): Record<SourceId, SourceMetadata>;
   getQuote(request: SourceListRequest): Promise<SourceListResponse>;
```

Comment:
I think we'll need to get rid of `getQuote`, and simply expose `getQuotes`. So that we can handle everything at the quote source list layer